cloud/linode/common.go

```diff
@@ -4,11 +4,16 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/linode/linodego"
 )
 
-const providerIDPrefix = "linode://"
+const (
+	providerIDPrefix = "linode://"
+	// context timeout for API requests
+	requestTimeout = 120 * time.Second
```

Comment:
```suggestion
	defaultRequestTimeout = 120 * time.Second
```
This should be the default if no timeout is configured, but I'm not sure if it should be a flag or env var, I have no strong opinion either way.